code/testapps/testapp/src/main/AndroidManifest.xml

```diff
@@ -55,14 +55,14 @@
         <activity android:name=".SignUpActivity"></activity>
         <activity android:name=".SettingsActivity"></activity>
 
-        <receiver android:name="com.adobe.marketing.mobile.campaign.LocalNotificationHandler"  android:exported="true">
+        <receiver android:name="com.adobe.marketing.mobile.LocalNotificationHandler"  android:exported="true">
             <intent-filter>
                 <action android:name="android.intent.action.BOOT_COMPLETED"/>
                 <action android:name="android.intent.action.INPUT_METHOD_CHANGED" />
             </intent-filter>
         </receiver>
 
-        <receiver android:name="com.adobe.marketing.mobile.campaign.NotificationDismissalHandler" />
+        <receiver android:name="com.adobe.marketing.mobile.NotificationDismissalHandler" />
 
```

Comment:
Let's make a note to add this in the sdk docs

Reply:
Yes, I have added this to the 3.x migration wiki doc.